styles.css

```
@@ -0,0 +1,1985 @@
+*,
+*::before,
+*::after {
+    box-sizing: border-box;
+}
+
+.logo {
+    background-color: transparent;
+    height: 239px;
+    width: 240px;
+    margin-left: 0px;
+    margin-right: 0px;
+    padding-bottom: 0;
+    position: relative;
+    left: 65px;
+    right: 48px;
+}
+
+h1 {
+    color: black;
+    font-family: 'Teko', sans-serif;
+    font-size: 48px;
+    margin-bottom: 30px;
+    margin-top: 20px;
+    max-width: 375px;
+    line-height: 45px;
+    text-align: center;
+}
+
+h2 {
+    color: #163346;
+    font-family: 'Teko', sans-serif;
+    font-size: 26px;
+    font-weight: 400;
+    height: 60px;
+    margin-bottom: 57px;
+    margin-top: 20px;
+    max-width: 375px;
+    text-align: center;
+}
+
+h3 {
+    border-bottom: 4px solid #163346;
+    display: block;
+    font-family: 'Teko', sans-serif;
+    font-size: 40px;
+    height: 50px;
+    margin-bottom: 20px;
+    margin-top: 0px;
+    letter-spacing: 1px;
+    width: 140px;
+    text-align: left;
+}
+
+.character {
+    border-bottom: 4px solid #163346;
+    color: #eaecf0;
+    font-family: 'Teko', sans-serif;
+    font-size: 40px;
+    font-weight: 300px;
+    letter-spacing: 1px;
+    line-height: 40px;
+    margin-bottom: 25px;
+    margin-top: 50px;
+    width: 225px;
+}
+
+body {
+    background-color: #d1d5de;
+    font-family: 'BenchNine', sans-serif;
+    font-weight: 700;
+    max-width: 375px;
+}
+
+.button-reset {
+    background: #6bf99c;
+    border: 6px;
+    cursor: pointer;
+    display: block;
+    font-family: 'Teko', sans-serif;
+    font-size: 32px;
+    font-weight: 700;
+    left: 35px;
+    right: 61px;
+    height: 87px;
+    margin-top: 45px;
+    padding-bottom: 10px;
+    padding-left: 21px;
+    padding-right: 20px;
+    padding-top: 15px;
+    position: relative;
+    text-align: center;
+}
+
+​ .button-reset[disabled] {
+    cursor: default;
+}
+
+.button-reset:hover {
+    background: #fff;
+    border: 6px solid #6bf99c;
+    color: #000;
+    padding-bottom: 1px;
+    padding-left: 12px;
+    padding-right: 11px;
+    padding-top: 9px;
+
+}
+
+input,
+select,
+textarea {
+    background-color: #fff;
+    font-family: inherit;
+    font-size: inherit;
+}
+
+select {
+    cursor: pointer;
+}
+
+textarea {
+    /*height: 50px;*/
+    /*width: 275px;*/
+
+}
+
+fieldset {
+    border: 0;
+    padding: 0px;
+    /*vertical-align: top;*/
+}
+
+fieldset label {
+    bottom: 8px;
+    position: relative;
+    bottom: 50px;
+    left: 37px;
+}
+
+fieldset span {
+    display: block;
+}
+
+fieldset+fieldset {
+    margin-top: 34px;
+}
+
+input[type="radio"],
+input[type="checkbox"] {
+    -moz-appearance: none;
+    -webkit-appearance: none;
+    appearance: none;
+    background-color: #1e2019;
+    border: 3px solid #d1d5de;
+    height: 24px;
+    width: 24px;
+}
+
+input[type="radio"]:focus,
+input[type="checkbox"]:focus {
+    border: 3px solid #6bf99c;
```

All that changes on focus is the border color, so your rule should be specific to that:
```suggestion
    border-color: #6bf99c;
```

This helps in two ways:
1. It makes it easier to tell at a glance _what_ changes on hover, focus or even in a media query.
1. It makes your CSS less brittle -- if you decided the `border-width` here needed to be thinner or wider, you'd have to remember to change it in two places. That's less than ideal and can cause unintentional errors/mistakes.